go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/cometbft/cometbft-db v0.7.0
 	github.com/cosmos/cosmos-proto v1.0.0-beta.3
 	github.com/cosmos/cosmos-sdk v0.46.15
+	github.com/cosmos/ibc-apps/middleware/packet-forward-middleware/v6 v6.1.2-0.20240228222509-2b38263c41d3
```

Comment:
@faddat can't find this commit /tag on the repo
why not using the tagged `v6.1.1`?

Reply:
We have a slack channel set up and it's probably best to discuss this there.